packages/react-native-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -142,6 +142,53 @@ extension RCTLogLevel {
     }
 }
 
+// Definition of JavaScript exception, which will be used to
+// log exception to the Crash Logging service.
+public struct GutenbergJSException {
+    public let type: String
+    public let value: String
+    public let stacktrace: [StacktraceLine]
+    public let context: [String: Any]
+    public let tags: [String: String]
+    public let isHandled: Bool
+    public let handledBy: String
+
+    public struct StacktraceLine {
+        public let filename: String?
+        public let function: String?
+        public let lineno: NSNumber?
+        public let colno: NSNumber?
+        
+        init(from dict: [AnyHashable: Any]) {
+            self.filename = dict["filename"] as? String
+            self.function = dict["function"] as? String
+            self.lineno = dict["lineno"] as? NSNumber
+            self.colno = dict["colno"] as? NSNumber
```

Comment:
Is it possible that all of these values are nil? Can we treat them as non-optional like the `GutenbergJSException.init?(from:)` function below?

Reply:
Good observation. I realized that `function` will be always defined as for undefined values it uses `?`. I'll update the function.

https://github.com/WordPress/gutenberg/blob/c017e122aa58dbfcc5e33fad72a1b9207ac77880/packages/react-native-bridge/lib/parseException.js#L56

https://github.com/WordPress/gutenberg/blob/c017e122aa58dbfcc5e33fad72a1b9207ac77880/packages/react-native-bridge/lib/parseException.js#L42

Reply:
Change applied in https://github.com/WordPress/gutenberg/pull/59221/commits/7c18727c5dd0412d926fc4456b4e585b063fee9d.